sensor_msgs/Image rgb_image
sensor_msgs/Image depth_image
float64[9]  cam_K_matrix
---
sensor_msgs/PointCloud2[] objects_clouds
geometry_msgs/Pose[] grasps
image_agnostic_segmentation/ImagePixels objects_pixels
